src/gpu_sysman.c

```diff
@@ -1644,13 +1658,15 @@ static bool gpu_freqs_throttle(gpu_device_t *gpu) {
   metric_family_t fam_ratio = {
       .help =
           "Ratio (0-1) of HW frequency being throttled during query interval",
-      .name = METRIC_PREFIX "throttled_ratio",
+      .name = METRIC_PREFIX "throttled",
       .type = METRIC_TYPE_GAUGE,
+      .unit = "1",
   };
   metric_family_t fam_counter = {
       .help = "Total time HW frequency has been throttled (in microseconds)",
-      .name = METRIC_PREFIX "throttled_usecs_total",
+      .name = METRIC_PREFIX "throttled.time",
       .type = METRIC_TYPE_COUNTER,
+      .unit = "us",
   };
```

Comment:
Aren't these two metrics reporting the same thing, just scaled by 1,000,000?

Reply:
They are for different time intervals / use-cases.  One is total time since (collectd) startup, which will be (soon) changed to be reported in seconds, and the other is ratio of time device was throttled between previous and current query.

Counter is more useful for things like Prometheus, ratio for direct monitoring.  I do not not expect both to be enabled at the same time.

Reply:
As mentioned above, I think we should consider support for counters to be ubiquitous and not make design decisions based on debugging output.

Reply:
This PR is about converting current Sysman plugin metric units & names to OTEL spec, not about changing plugin metric variant handling.  That discussion belonged to #4047 and #4173 (and maybe bit to #4177), not this one.

Maybe you could file a ticket where we can discuss what should be done about the variants?

Reply:
Note that while metrics output directly from plugin is useful for debugging collectd, that's not its main use-case, but the local GPU monitoring (I'll add note of that in #4177).